src/locales/de/translations.json

```diff
@@ -103,13 +103,15 @@
   "career.leadbox.subtitle": "Initiativbewerbung",
   "career.leadbox.text": "Georgios Kaleadis, CTO",
   "career.leadbox.mail": "career@satellytes.com",
-  "office.title": "Office",
-  "office.heading": "Unser neues Office direkt am Sendlinger Tor",
-  "office.team.title": "Team",
-  "office.team.heading": "Das sind wir",
-  "office.team.text": "Unser erfahrenes Team aus Programmierern und Designern",
-  "office.leadbox.title": "Haben Sie Interesse bekommen?",
-  "office.leadbox.link": "Karriere",
+  "about-us.title": "Über Uns",
+  "about-us.office.title": "Office",
+  "about-us.office.heading": "Unser neues Office direkt am Sendlinger Tor",
+  "about-us.office.text": "Fühlen Sie sich bei uns wie zu Hause. Ein paar kleine Eindrücke unseres neuen Büros in der Sendlinger Straße im Herzen Münchens.",
+  "about-us.team.title": "Team",
+  "about-us.team.heading": "Das sind wir",
+  "about-us.team.text": "Unser erfahrenes Team aus Programmierern und Designern",
+  "about-us.leadbox.title": "Haben Sie Interesse bekommen?",
+  "about-us.leadbox.link": "Karriere",
```

Comment:
I'm not happy about the dash but I wonder if an underscore would help much here. Technically fine so thanks!